scripts/compile-aggregated-java-sources.sh

```diff
@@ -21,6 +21,8 @@ mvnget() {
 mvnget org/apache/ant/ant/1.7.0/ant-1.7.0.jar
 mvnget com/google/auto/value/auto-value-annotations/1.7/auto-value-annotations-1.7.jar
 mvnget javax/annotation/javax.annotation-api/1.3.2/javax.annotation-api-1.3.2.jar
+mvnget org/openjdk/jmh/jmh-core-1.22.jar
```

Comment:
The compile-aggregated-java-sources is used to compile jflex outside with generated code. No tests are executed in this branch. I don't think we want  to run a benchmark in this branch. The Maven project should be sufficient.

Reply:
Good point. We should probably just remove the benchmark directory for that one. I'm about to head off out of phone coverage now. Feel free to adjust or leave to me for later.

Reply:
Now done.